extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/KafkaIndexTaskTest.java

```diff
@@ -3095,7 +3095,14 @@ private void makeToolboxFactory() throws IOException
         testUtils.getTestObjectMapper(),
         derby.metadataTablesConfigSupplier().get(),
         derbyConnector
-    );
+    )
+    {
+      @Override
+      public int getSqlMetadataMaxRetry()
+      {
+        return 2;
```

Comment:
Adding only 2 retries here, so tests will not take much time to finish. The retries are 10 by default, and tests take 8-10 minutes to complete.

Reply:
can we please add a comment in the code to explain why we are overriding this method?